app/main.py

```diff
@@ -1 +1,43 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    @classmethod
+    def cleanup_dead_animals(cls) -> None:
+        cls.alive = [animal for animal in cls.alive if animal.health > 0]
+
+    def hide(self) -> None:
+        self.hidden = not self.hidden
+
+    def __str__(self) -> str:
+        return f"{{Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}}}"
```

Comment:
The `__str__` method should be adjusted to return a dictionary format instead of a string. The task requires that printing `Animal.alive` should show a list of dictionaries.